### PR TITLE
Rails Docs - updates to props

### PR DESCRIPTION
**Pull request by QuintonJason**
## Description

- [x] - updated docs with any missing properties
- [x] - alphabetized the docs and component files
### Screenshots

There are no visual differences

### Steps for testing
1. Check pages and verify. 

### QA Steps
(Low) #330 Documentation updates to standardize, clean up, and add missing props for elements and objects.

### CODE REVIEW

File: docs/app/views/examples/elements/choice/_props.html.erb [hunk 1]
```diff
@@ -9,28 +9,39 @@
...
+  <td><%= md('`attributes`') %></td>
+  <td><%= md('For setting additional attributes not defined above. Accepts a Ruby hash of valid key-value properties, such as data-attributes') %></td>
+  <td><%= md('Hash') %></td>
+  <td><%= md('`nil`') %></td>
 </tr>
 <tr>
-  <td><%= md('`text`') %></td>
-  <td><%= md('Sets the content of the primary text line') %></td>
+  <td><%= md('`disabled`') %></td>
+  <td><%= md('Sets the state to \"disabled\" by default.') %></td>
```
Comment (monkeypox8) [inline docs/app/views/examples/elements/choice/_props.html.erb:20]: Can remove the escape `\` here

File: docs/app/views/examples/objects/feature_toggle/_props.html.erb [hunk 2]
```diff
@@ -18,8 +18,17 @@
...
-  <td><%= md('Array') %></td>
+  <td><%= md('An array of optional links.') %></td>
+  <td>
+    <%= md('```
+Array<{
+  icon: String,
+  location: String,
+  target: String,
+  text: String
+}>
```
Comment (monkeypox8) [inline docs/app/views/examples/objects/feature_toggle/_props.html.erb:29]: Is it okay that the closing tick marks are missing here and a few other files? Seems to render just fine without them, so I'd assume the closing markdown tag takes care of it 🤷🏼‍♂️

File: docs/app/views/examples/objects/input_group/_props.html.erb [hunk 3]
```diff
@@ -76,3 +58,21 @@
   <td><%= md('String: ["small", "large"]') %></td>
   <td><%= md('`nil`') %></td>
 </tr>
+<tr>
+  <td><%= md('`group_button_type`') %></td>
+  <td><%= md('This can be any allowed value from the list of types specified for the <a href=\"#element-form_input\" rel=\"nofollow\">input element</a>.') %></td>
```
Comment (monkeypox8) [inline docs/app/views/examples/objects/input_group/_props.html.erb:63]: Can remove the escape characters `\` here

File: docs/app/views/examples/objects/upload_card/_props.html.erb [hunk 4]
```diff
@@ -23,27 +36,14 @@
...
-  <td><%= md('`errors`') %></td>
-  <td><%= md('Array containing error messages') %></td>
-  <td>
-    <%= md('```
-      Array<{
-        text: String,
-      }>
-    ') %>
-  </td>
+  <td><%= md('`selection_label`') %></td>
```
Comment (monkeypox8) [inline docs/app/views/examples/objects/upload_card/_props.html.erb:45]: Should `selection_label` be above `selection_subtext` in the order?

**Review by monkeypox8 — APPROVED** [review]
👍🏼 🔥  LGTM! Added a couple of nitpicky ( & non-blocking) items to check out. Thanks for taking on this monster.

**Review by joedakroub — COMMENTED** [review]
Running this locally, everything but the `Typeahead` looks great!

http://0.0.0.0:4000/pages/object/typeahead

Screen Shot 2021-02-23 at 11 43 40 AM

Not related to your work, but possibly an easy fix?

**Review by philschanely — APPROVED** [review]
No concerns here.

**Comment by QuintonJason** [comment]
> Running this locally, everything but the `Typeahead` looks great!
> 
> http://0.0.0.0:4000/pages/object/typeahead
> 
> Screen Shot 2021-02-23 at 11 43 40 AM
> 
> Not related to your work, but possibly an easy fix?

I'll throw up a filler page, good catch Joe!

File: docs/app/views/examples/objects/feature_toggle/_props.html.erb (hunk 2 above)
Comment (QuintonJason) [reply docs/app/views/examples/objects/feature_toggle/_props.html.erb:29]: When I added the ticks, they were visible in the code block. As a result, I omitted.

**Pull request merged**